packages/snap/src/index.ts

```diff
@@ -120,8 +120,18 @@ export const onRpcRequest: OnRpcRequestHandler = async ({
               heading('Identity info'),
               divider(),
               text('Private key:'),
-              copyable(identity.privateKeyHex),
+              text(
+                'Save your private key to recover your account if you lose access',
```

Comment:
Please save your private key and keep it safe. The loss of the private key will lead to the loss of access to your profile!